ufr-tours-histoire-de-lart-note.csl

```diff
@@ -0,0 +1,662 @@
+            <if type="motion_picture" match="any">
+              <date date-parts="year" form="text" variable="issued">
+                <date-part name="year"/>
+              </date>
+            </if>
+            <else-if type="broadcast" match="any">
+              <group delimiter=" ">
+                <text value="diffusé le"/>
+                <date form="text" variable="issued">
+                  <date-part name="day"/>
+                  <date-part name="month" form="short"/>
+                  <date-part name="year"/>
+                </date>
+              </group>
+            </else-if>
+          </choose>
+          <group delimiter=" ">
+            <text term="available at" suffix=" :"/>
+            <text variable="URL"/>
+          </group>
+          <group delimiter=" " prefix="[" suffix="]">
+            <text term="accessed"/>
+            <date form="text" variable="accessed">
+              <date-part name="day"/>
+              <date-part name="month"/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if type="bill book graphic legal_case motion_picture report song chapter paper-conference" match="any">
+        <choose>
+          <if is-numeric="edition">
+            <group delimiter=" ">
+              <number variable="edition" form="ordinal"/>
+              <text term="edition" form="short"/>
+            </group>
+          </if>
+          <else>
+            <text variable="edition"/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <group font-style="normal" delimiter=", ">
+          <choose>
+            <if variable="issued">
+              <text macro="volume"/>
+              <text macro="issue"/>
+              <date form="text" variable="issued">
+                <date-part name="day"/>
+                <date-part name="month"/>
+                <date-part name="year"/>
+              </date>
+            </if>
+            <else>
+              <text macro="volume" text-case="capitalize-first"/>
+            </else>
+          </choose>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="volume">
+    <choose>
+      <if is-numeric="volume">
+        <text term="volume" form="short" suffix=". "/>
+        <text variable="volume"/>
+      </if>
+      <else>
+        <text variable="volume"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if match="all" variable="issue">
+        <text term="issue" form="short" suffix=" "/>
+        <text variable="issue"/>
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <text variable="collection-title" quotes="true" prefix="coll. "/>
+  </macro>
+  <citation et-al-min="4">
```

Comment:
@Emic37, we don't allow `et-al-min` to be used without setting `et-al-use-first` on the same element.